src/handlers/categoria.py

```diff
@@ -8,7 +8,5 @@
 def categoria(update: Update, context: CallbackContext) -> None:
     if 'categoria' in context.user_data:
         del context.user_data['categoria']
-    reply_markup = ReplyKeyboardMarkup(
-        [[c] for c in categorie], resize_keyboard=True, one_time_keyboard=True)
-    context.bot.sendMessage(chat_id=update.message.chat_id,
-                            text="Scegli la categoria delle domande:", reply_markup=reply_markup)
+    reply_markup = ReplyKeyboardMarkup([[c] for c in categorie], resize_keyboard=True, one_time_keyboard=True)
```

Comment:
rendi il codice più leggibile, i cicli for "inline" non sono tanto leggibili, personalmente preferisco estrarli e "wrapparli" in una variabile separata
```suggestion
    categories_reply_keyboard = [[c] for c in categorie]
    reply_markup = ReplyKeyboardMarkup(categories_reply_keyboard, resize_keyboard=True, one_time_keyboard=True)
```